ta/ocall/include/user_ta_header_defines.h

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2020, Microsoft Corporation
+ */
+
+#ifndef USER_TA_HEADER_DEFINES_H
+#define USER_TA_HEADER_DEFINES_H
+
+#include <ta_ocall_test.h>
+#include <user_ta_header.h>
+
+#define TA_UUID		TA_OCALL_UUID
+
+#define TA_FLAGS	(TA_FLAG_SINGLE_INSTANCE | \
+			 TA_FLAG_MULTI_SESSION | \
```

Comment:
The `MULTI_SESSION` flag should not be needed. I mean, I know it is, due to the way the test is written (I tried running without and `test_ocall_during_session_open()` fails to open a session since the one initially created for the param tests is not closed yet).
I would recommend changing `xtest_tee_test_1100()` slightly or splitting it in two so that we can make sure OCALLs work in single instance single session TAs.

Reply:
I'll work on this next.